imports/plugins/included/marketplace/client/templates/stripeConnectSignupButton/stripeConnectSignupButton.js

```diff
@@ -41,17 +41,25 @@ Template.stripeConnectSignupButton.events({
       return Alerts.toast(`${i18next.t("admin.connect.shopAddressNotConfigured")}`, "error");
     }
 
-    const email = shop.emails[0].address;
```

Comment:
the business form does not have an email input, so we grab the user's default email address instead